qt-models/cylindermodel.cpp

```diff
@@ -233,11 +233,8 @@ QVariant CylindersModel::data(const QModelIndex &index, int role) const
 		if (index.column() == REMOVE) {
 			same_gas = same_gasmix_cylinder(cyl, index.row(), &displayed_dive, false);
 
-			if ((in_planner() || same_gas == -1) &&
-				((DivePlannerPointsModel::instance()->currentMode() != DivePlannerPointsModel::NOTHING &&
```

Comment:
I have to admit, I am a bit lost. What was this currentMode() != ...:NOTHING and why is it not necessary anymore? A slightly more verbose commit message would be helpful.

Reply:
currentMode() != ...:NOTHING is IMHO equivalent to in_planner() but in_planner() is much more familiar to everyone.
This is why I replaced it.